app/js/question_controller.js

```diff
@@ -109,31 +139,49 @@
         answers[i].created = Utils.time_since(new Date(created));
       }
 
+      add_thread_header(question);
       var html = nunjucks.render('thread.html', {
+        user: user,
         results: answers
       });
-      document.getElementById('comment-list').innerHTML = html;
+      var list = document.getElementById('comment-list');
+      list.insertAdjacentHTML('beforeend', html);
     });
   }
 
   var QuestionController = {
     init: function() {
-      var form = document.getElementById('question_form');
-      form.addEventListener('submit', submit_comment);
+      var question_view = location.search ? true : false;
+
+      // if this is a question view, not arrived at from clicking/tapping
+      // ask a question, hide the introduction section.
+      if (question_view) {
+        document.getElementById('thread-introduction').classList.add('hide');
+      }
```

Comment:
I will have a look but, I remember that after the changes there was a quick flash of the introduction view if this was not done here. I will test again however and change if that is no longer the case.

I believe it was related to the fact the loading the user takes a little time so...